### DISCUSSION TITLE
no type named 'type' in 'struct std::enable_if<false, void>'  ostream failure

### DISCUSSION THREAD

**Discussion by Gruby47** [community/community#78023] [Programming Help]
### Select Topic Area

Question

### Feature Area

Issues

### Body

no type named 'type' in 'struct std::enable_if<false, void>'

this communicate popped up on my terminal. i had similar problem with istream earlier but i managed it. Now this happened but it's not the same solution i used with istream. does somebody know how to fix this? PLS help me because im clueless 

![obraz_2023-12-06_145310310](https://github.com/community/community/assets/146835409/5313fdb5-9fc9-45f7-bf8a-f8c94d70fdd4)

![image](https://github.com/community/community/assets/146835409/5156330c-0757-47a9-aa5a-7d28624a8ba0)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Gruby47 ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects. 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Comment by Nav45666** [comment]
I know this post is an year old.. but i have the same issue. were you able to find a way to resolve this issue?
